src/react/ui/SingleCardPage.tsx

```diff
@@ -21,7 +21,7 @@ import { Container } from 'react-bootstrap';
 import BackButton from './BackButton';
 
 interface Props {
-  backLink: string;
```

Comment:
i saw this line in many other PRs. you should make a separate commit for it so you will have an easier time when rebasing everything after a merge